crates/sui-framework-tests/src/lib.rs

```diff
@@ -70,6 +77,345 @@ mod test {
         });
     }
 
+    #[test]
+    #[cfg_attr(msim, ignore)]
+    fn test_metered_move_bytecode_verifier() {
```

Comment:
Tests moved from `move_package_tests` to avoid exposing `Scope` and `Meter` types via `sui-execution`.

Reply:
why not in its own file? I guess because `lib.rs` is the only file in this directory?

Reply:
Yeah, it's a fair question. I was just doing this mechanically, but a new file sounds good.